UI.py

```diff
@@ -237,26 +239,26 @@ def CreateGetLeavesAcrossRangeOptions():
 
     return member_options
 
-def CreateMultipleLeavesEmbed(leaves, include_reason):
+def CreateLeavesAcrossRangeEmbed (leaves, include_reason):
     embeds_to_send = []
     for leaves_group in leaves:
         embed = discord.Embed(
             title = f'Applied Leaves',
             description = f'<@!{leaves_group[0][0]["member_id"]}> has requested:',
             colour = 0x4682B4
         )
-        embed.set_thumbnail(url = os.getenv("Leave_Balance_Link"))
+        embed.set_thumbnail(url = os.getenv("Leave_Balance_Image"))
         embed.add_field(name = '\u200B', value = '\u200B', inline = False)
-        embed = DesignMultipleLeavesEmbed(leaves_group, embed, include_reason)
+        embed = FormatingLeavesAcrossRangeEmbed(leaves_group, embed, include_reason)
         if (embed == None):
             continue
 
-        footer_text = (("\u200B " * 150) + datetime.date.today().strftime("%d/%m/%Y")) # magic number 150
+        footer_text = (("\u200B " * embed_footer_spaces_count) + datetime.date.today().strftime("%d/%m/%Y"))
         embed.set_footer(text = footer_text)
         embeds_to_send.append(embed)
     return embeds_to_send
 
-def DesignMultipleLeavesEmbed(leaves_group, embed, include_reason):
+def FormatingLeavesAcrossRangeEmbed(leaves_group, embed, include_reason):
```

Comment:
```suggestion
def FormatLeavesAcrossRangeEmbed(leaves_group, embed, include_reason):
```
Verb function names.

Reply:
that was auto complete, completely my fault